scheduler/cloudformation/cloudformation.go

```diff
@@ -706,12 +708,29 @@ func (s *Scheduler) Instances(ctx context.Context, app string) ([]*scheduler.Ins
 			}
 			taskDefinitions[k] = resp.TaskDefinition
 		}
+
+		k = *t.ContainerInstanceArn
+
+		if _, ok := containerInstances[k]; !ok {
+			resp, err := s.ecs.DescribeContainerInstances(&ecs.DescribeContainerInstancesInput{
+				Cluster:            t.ClusterArn,
+				ContainerInstances: []*string{t.ContainerInstanceArn},
```

Comment:
We can optimize this better so we make less calls:

1. Iterate through the tasks, and add the container instance arn's to a map.
2. Chunk them to whatever the maximum number of container instances is for `DescribeContainerInstances`, and describe each chunk (the [docs](http://docs.aws.amazon.com/AmazonECS/latest/APIReference/API_DescribeContainerInstances.html) don't seem to specify what the limit is). Add that to another map, where the key is the container instance arn, and the value is the instance id.